### DISCUSSION TITLE
GEF reports invalid escape sequence when reaching breakpoints.

### DISCUSSION THREAD

**Discussion by TommyJerryMairo** [hugsy/gef#734] [Q&A] [answered]
```
/home/tjm/.config/gef/gef-915ff9114127a6c02301fb3faddb0976737269db.py:2469: DeprecationWarning: invalid escape sequence '\Ç'
  res = gdb.Value(address).cast(char_ptr).string(encoding=encoding, length=length).strip()
/home/tjm/.config/gef/gef-915ff9114127a6c02301fb3faddb0976737269db.py:2469: DeprecationWarning: invalid escape sequence '\ '
  res = gdb.Value(address).cast(char_ptr).string(encoding=encoding, length=length).strip()
```
[gef-915ff9114127a6c02301fb3faddb0976737269db.py.gz](https://github.com/hugsy/gef/files/7255032/gef-915ff9114127a6c02301fb3faddb0976737269db.py.gz)

I'm running GEF with GDB 11.1 and Python 3.9
```
GNU gdb (GDB) 11.1
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "x86_64-pc-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<https://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word"...
GEF for linux ready, type `gef' to start, `gef config' to configure
91 commands loaded for GDB 11.1 using Python engine 3.9
[*] 5 commands could not be loaded, run `gef missing` to know why.
```

**Comment by clark-ja** [comment, marked as answer]
Hello @TommyJerryMairo,

thank you for using GEF. This seems more like an issue than a discussion. Also you seem to be using a GEF version from July 2021, please update GEF to the latest `dev` branch and see if your issue still persists. If so please open an issue with a more detailed description when this problem occurs for you: What are you doing beforehand exactly and in what moments are the errors raised. Also please run `gef config gef.debug True` inside GEF before running the command in question for more detailed error messages and a stack trace. Furthermore always provide us with the output of the `version` command when you're having troubles. Thank You!

I am closing this discussion for now. Feel free to open an issue as described if you're still having the problems with the newest GEF `dev` branch.
